### PR TITLE
Add support for cross reference fields in Mappe and Registrering

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

**Review by rveda — APPROVED** [review]

File: src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java [hunk 1]
```diff
@@ -215,6 +219,46 @@ public void setMappetype(Mappetype mappetype) {
 		return link(NOEKKELORD_LINK, noekkelord);
 	}
 
+	public LinkAction<TEntity> addKryssReferanseTilMappe(String... mappeIds) {
```
Comment (teodoradobreva) [inline src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java]: Shouldn't `addKryssReferanseTilMappe` be changed to `addKryssreferanseTilMappe` as kryssreferanse is one word in all other places?

Applies to all added methods.

File: src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java [hunk 2]
```diff
@@ -215,6 +219,46 @@ public void setMappetype(Mappetype mappetype) {
...
+
+		return link(KRYSSREFERANSE_TIL_MAPPE_LINK, mappeIds);
+	}
+
+	public LinkAction<TEntity> addKryssReferanseTilMappe(AbstraktMappe... mappeIds) {
```
Comment (teodoradobreva) [inline src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java]: Parameter `mappeIds` should be changed to just `mappe` (?).

Applies to all methods that accept Noark objects.

Comment (ivaylomitrev) [reply src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java]: Fixed.

File: src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java (hunk 1 above)
Comment (ivaylomitrev) [reply src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java]: Yes. Fixed.

**Comment by ivaylomitrev** [comment]
I rebased on develop and force-pushed to resolve conflicts with the introduction of Merknad.

**Review by teodoradobreva — APPROVED** [review]

**Comment by ivaylomitrev** [comment]
Squashed and rebased on latest develop in preparation for merging.

**Pull request merged**